#!/usr/bin/env python
PACKAGE = "mars_simulation_ctv_agent"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("gain_kp", double_t, 0, "Proportional Gain", 1.5, 0)
gen.add("gain_ki", double_t, 0, "Integral Gain", 1.0, 0)
gen.add("gain_kd", double_t, 0, "Differential Gain", 0.4, 0)

exit(gen.generate(PACKAGE, "mars_simulation_ctv_agent", "dynamic_pid"))